packages/fuel-gauge/src/await-execution.test.ts

```diff
@@ -1,90 +1,90 @@
-import { Provider, WalletUnlocked, randomBytes, Wallet, FUEL_NETWORK_URL } from 'fuels';
-import { launchNode } from 'fuels/test-utils';
+import { Wallet } from 'fuels';
+import { launchTestNode } from 'fuels/test-utils';
 
 /**
  * @group node
+ * @group browser
  */
 describe('await-execution', () => {
   test('awaiting execution of a transaction on the provider works', async () => {
-    const { cleanup, ip, port } = await launchNode({
-      args: ['--poa-instant', 'false', '--poa-interval-period', '400ms'],
+    using launched = await launchTestNode({
+      nodeOptions: {
+        args: ['--poa-instant', 'false', '--poa-interval-period', '400ms'],
+      },
     });
-    const nodeProvider = await Provider.create(`http://${ip}:${port}/v1/graphql`);
-    const baseAssetId = nodeProvider.getBaseAssetId();
 
-    const genesisWallet = new WalletUnlocked(
-      process.env.GENESIS_SECRET || randomBytes(32),
-      nodeProvider
-    );
+    const {
+      provider,
+      wallets: [genesisWallet],
+    } = launched;
 
-    const destination = Wallet.generate({ provider: nodeProvider });
+    const destination = Wallet.generate({ provider });
 
-    const transfer = await genesisWallet.createTransfer(destination.address, 100, baseAssetId, {
-      gasLimit: 10_000,
-    });
+    const transfer = await genesisWallet.createTransfer(
+      destination.address,
+      100,
+      provider.getBaseAssetId(),
+      {
+        gasLimit: 10_000,
+      }
+    );
 
     transfer.updateWitnessByOwner(
       genesisWallet.address,
       await genesisWallet.signTransaction(transfer)
     );
 
-    const response = await nodeProvider.sendTransaction(transfer, { awaitExecution: true });
+    const response = await provider.sendTransaction(transfer, { awaitExecution: true });
 
     expect(response.gqlTransaction?.status?.type).toBe('SuccessStatus');
-
-    cleanup();
   });
 
   test.skip('transferring funds with awaitExecution works', async () => {
-    const provider = await Provider.create(FUEL_NETWORK_URL);
-    const baseAssetId = provider.getBaseAssetId();
+    using launched = await launchTestNode({
+      walletsConfig: {
+        amountPerCoin: 1_000_000,
+      },
+    });
```

Comment:
```suggestion
    using launched = await launchTestNode();
```

[Default](https://github.com/FuelLabs/fuels-ts/blob/7ae80b8a4af7a2b770bb687edf1d9db671cc4a60/packages/account/src/test-utils/setup-test-provider-and-wallets.ts#L33) `amountPerCoin` is `10_000_000_000` so you don't need to define this.

Reply:
755f8e04b